deploy/docker/server.py

```
@@ -33,14 +34,14 @@
 )
 
 from utils import (
-    FilterType, load_config, setup_logging, verify_email_domain
+    FilterType, combine_lifespans, load_config, setup_logging, verify_email_domain
```

Ruff (0.11.9)

`utils.FilterType` imported but unused

Remove unused import: `utils.FilterType`

(F401)